app/controllers/change_requests_controller.rb

```diff
@@ -233,7 +305,12 @@ def get_field_change_hash(change_request)
   end
 
   def field_changes
-    params[:change_request].to_unsafe_h.map do |name, value|
+    if params[:change_request][:field_changes]
+     fields = params[:change_request][:field_changes]
```

Comment:
```suggestion
      fields = params[:change_request][:field_changes]
```

Could you indent this by one more space to make it consistent with everything else?

Also, I'm honestly pretty surprised that Rubocop doesn't complain about this, given how much it complains about other stuff...